tests/framework/installer/install_data.go

```diff
@@ -312,3 +324,31 @@ spec:
         - key: data
           path: mon-endpoints`
 }
+
+//GetCleanupPod gets a cleanup Pod manifest
+func (i *InstallData) GetCleanupPod(removalDir string) string {
+	return `apiVersion: batch/v1
+kind: Job
```

Comment:
good idea to cleanup. it would be nice to have a sample job like this in the `cluster/examples/kubernetes` folder that will clean up their dataDirHostPath for them on all machines with a daemonset. (separate PR of course)

Reply:
will do